libcxx/docs/FeatureTestMacroTable.rst

```diff
@@ -266,7 +266,7 @@ Status
     --------------------------------------------------- -----------------
     ``__cpp_lib_polymorphic_allocator``                 ``201902L``
     --------------------------------------------------- -----------------
-    ``__cpp_lib_ranges``                                ``202207L``
+    ``__cpp_lib_ranges``                                ``202110L``
```

Comment:
do we have a test for this change? I wonder what is the reason to combine https://github.com/llvm/llvm-project/issues/70557 with implementing this paper?

Reply:
>I wonder what is the reason to combine https://github.com/llvm/llvm-project/issues/70557 with implementing this paper?

P2602 bumps `__cpp_lib_ranges`, so I wanted to address issue with its value. I'm going to introduce this change in separate PR though.

Commit: https://github.com/llvm/llvm-project/pull/74534/commits/a1dbe7d83476a1dd18f883c0340977806dab572f